substrate/frame/ranked-collective/src/benchmarking.rs

```diff
@@ -173,5 +173,20 @@ benchmarks_instance_pallet! {
 		assert_eq!(Voting::<T, I>::iter().count(), 0);
 	}
 
+	exchange_member {
+		let who = make_member::<T, I>(1);
+		let who_lookup = T::Lookup::unlookup(who.clone());
+		let new_who = account::<T::AccountId>("new-member", 0, SEED);
+		let new_who_lookup = T::Lookup::unlookup(new_who.clone());
+		let origin =
+			T::ExchangeOrigin::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
+		let call = Call::<T, I>::exchange_member { who: who_lookup, new_who:new_who_lookup};
+	}: { call.dispatch_bypass_filter(origin)? }
```

Comment:
Can you not use the underscore syntax like in the benchmark above?

Reply:
I don't seem to comprehend what you mean @ggwpez 
can you help me understand?

Reply:
explained in dm